### DISCUSSION TITLE
Control+Click ListboxItem deselects ListBoxItem

### DISCUSSION THREAD

**Discussion by tkefauver** [AvaloniaUI/Avalonia#12435] [Q&A]
**Describe the bug**
In a `ListBox` with `SelectionMode=Single`, Control+Click on a selected `ListBoxItem` will deselect the item.

**To Reproduce**

1. Run ControlCatalog.NetCore
2. Select the ListBox page
3. Select Item 4
4. Control+Click Item 4

**Expected behavior**
Control+Click is ignored/unhandled and selection is not changed.

**Desktop (please complete the following information):**
- OS: Windows 11
- Version 22H2

**Additional context**
I know this is by design but is there a way to change it? In my case there's hyperlinks within the `ListBoxItem` I want to restrict navigation to Control+Click but instead the item is deselected. As a workaround I'm using the Alt key but that's kind of unconventional. Is there a way to suppress the deselect?

**Comment by timunie** [comment]
> I know this is by design but is there a way to change it? In my case there's hyperlinks within the ListBoxItem I want to restrict navigation to Control+Click but instead the item is deselected. As a workaround I'm using the Alt key but that's kind of unconventional. Is there a way to suppress the deselect?

In this case: Please do not open an issue but a Q&A or join us on Telegram. Thank you. 

I'll convert it for you this time.

**Comment by timunie** [comment]
this should do the trick: 

```xml
<ListBox ItemsSource="{Binding MyItems}" 
         SelectionMode="AlwaysSelected, Single"/>
```

**Comment by tkefauver** [comment]
Nope I tried that, it still deselects the item then selects the first item. This seems like an 'Extended' selection mode kind of behavior (like explorer on windows, it was a mode in WPF) but not in single select mode.
